Connector/eth/apirpc.py

```diff
@@ -666,3 +667,88 @@ def call(id, params, config):
         )
 
     return response
+
+
+@rpcmethod.rpcMethod(coin=COIN_SYMBOL)
+@httpmethod.postHttpMethod(coin=COIN_SYMBOL)
+def getAddressHistory(id, params, config):
+
+    logger.printInfo(
+        f"Executing RPC method getAddressHistory with id {id} and params {params}")
+
+    requestSchema, responseSchema = utils.getMethodSchemas(GET_ADDRESS_HISTORY)
+
+    err = httputils.validateJSONSchema(params, requestSchema)
+    if err is not None:
+        raise error.RpcBadRequestError(
+            id=id,
+            message=err.message
+        )
+
+    try:
+        addrHistory = HTTPConnector.get(
+            endpoint=f"{config.indexerEndpoint}",
+            path=INDEXER_TXS_PATH,
+            params={
+                "and": f"(contract_to.eq.,or(txfrom.eq.{params['address']},txto.eq.{params['address']}))",
```

Comment:
Could be possible to send something like '*' instead of a valid address for creating a really heavy query? I'm imaging because I'm not used Postrest never before